pkg/artifactory/resource/repository/federated/federated.go

```diff
@@ -27,5 +27,7 @@ var FederatedRepoTypesSupported = []string{
 	"pypi",
 	"rpm",
 	"sbt",
+	"terraform_module",
+	"terraform_provider",
```

Comment:
These are added to the slice but there are no corresponding types in `schemaRepoTypeLookup` in `local.go` for the lookup.

Reply:
added